synapse/lib/stormtypes.py

```diff
@@ -6319,6 +6319,33 @@ class Layer(Prim):
                        'default': None, },
                   ),
                   'returns': {'type': 'int', 'desc': 'The count of rows.', }}},
+        {'name': 'getPropValueCount',
```

Comment:
Do we want this to be separate from `getPropCount` or should we just have `valu` be an optional arg? Currently retrieving the count with a value will be much faster than the scan `getPropCount` does, but in 3.0.0 they will both be fast so we could potentially merge them then?

Reply:
changed to make valu an optional arg on count APIs